### PR TITLE
docs: a11y: Add alt text to images on front page

### PR DESCRIPTION
**Pull request by julianna-ciq**
In this PR, I:
- [x] Added empty alt text to icons
- [x] Removed the alt text "Features" from an icon
- [x] Added alt text to the Twitter and Github links in the footer
- [x] Added alt text "FDC3" to the logo image
- [x] Added alt text "FDC3 1.2/2.0 Certified" to those badges

If you want to test the impact of these changes, here's how you can quickly see the difference using Chrome:
1. Go to the webpage (in Chrome)
2. Open Developer Tools (Win: press F12)
3. In DevTools, go to the "Lighthouse" tab
4. Click "Analyze page load" button

Before this PR, you'll see a score in the 80's, with an indicator that there are several images without alt text
image
**Lighthouse score for fdc3.finos.org**

After this PR, you'll see a score in the 90's, without any indicators about alt text
image
**Lighthouse score for dev version of the website, based on this PR**

### CODE REVIEW

File: website/data/features/api.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<img alt="Features" src="/img/feature-api.svg" />
+<img alt="" src="/img/feature-api.svg" />
```
Comment (kriswest) [inline website/data/features/api.mdx:1]: Is there any reason to not have actual alt text for these icons? They should probably also be links to the relevant sections

```suggestion
<a href="/docs/api/spec"><img alt="FDC3 API Part" src="/img/feature-api.svg" /></a>
```

File: website/data/features/app-directory.mdx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<img src="/img/feature-appd.svg" />
+<img alt="" src="/img/feature-appd.svg" />
```
Comment (kriswest) [inline website/data/features/app-directory.mdx:1]:
```suggestion
<a href="docs/app-directory/overview"><img alt="FDC3 App Directory Part" src="/img/feature-appd.svg" /></a>
```

File: website/data/features/context-data.mdx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<img src="/img/feature-context.svg" />
+<img alt="" src="/img/feature-context.svg" />
```
Comment (kriswest) [inline website/data/features/context-data.mdx:1]:
```suggestion
<a href="/docs/context/spec"><img alt="FDC3 Context Part" src="/img/feature-context.svg" /></a>
```

File: website/data/features/intents.mdx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<img src="/img/feature-intents.svg" />
+<img alt="" src="/img/feature-intents.svg" />
```
Comment (kriswest) [inline website/data/features/intents.mdx:1]:
```suggestion
<a href="/docs/intents/spec"><img alt="FDC3 Intents Part" src="/img/feature-intents.svg" /></a>
```

File: website/docusaurus.config.js [hunk 5]
```diff
@@ -174,10 +174,10 @@ module.exports={
           "title": "More",
           "items": [
             {
-              "html": "<a href=\"https://github.com/finos/FDC3\"><img src=\"https://img.shields.io/github/stars/finos/FDC3?label=FDC3&style=social\" /></a>"
+              "html": "<a href=\"https://github.com/finos/FDC3\"><img alt=\"Github\" src=\"https://img.shields.io/github/stars/finos/FDC3?label=FDC3&style=social\" /></a>"
             },
             {
-              "html": "<a href=\"https://twitter.com/FDC3_\"><img src=\"https://img.shields.io/twitter/follow/FDC3_?style=social\" ></a>"
+              "html": "<a href=\"https://twitter.com/FDC3_\"><img alt=\"Twitter\" src=\"https://img.shields.io/twitter/follow/FDC3_?style=social\" ></a>"
             },
```
Comment (kriswest) [inline website/docusaurus.config.js:181]: Lets just remove this whole entry - I don't believe the maintainers have access to this twitter account and theres next to nothing on it. It needs to go.

**Review by kriswest — COMMENTED** [review]
I've got a question and a few suggestions before this goes in @julianna-ciq

**Comment by julianna-ciq** [comment]
Generally, we want to use alt text when necessary, but only when necessary.

Take this block of content:
image

We have an image, what visually looks like a heading/link ("API"), and then a description about the API. In this case, the image's only purpose is to provide some kind of visual interest. If you didn't have the image, there would be no change in information. In this case, this image is "decorative", and we don't want to provide alt text for it.

**Review by kriswest — APPROVED** [review]
LGTM

**Pull request merged**